ui/src/shared/utils/options.ts

```diff
@@ -0,0 +1,140 @@
+import {Package, File, Expression, ObjectExpression} from 'src/types/ast'
+
+/*
+  Helper functions for editing option statements and object expressions
+  from the AST of a flux program.
+
+    option v = {a: 5, b: 6} // declaring an option in flux
+
+  After parsing a flux program via wasm and getting an AST representation,
+  it is sometimes useful/necessary to edit the options that are defined.
+  The functions in this file provide an interface for getting, setting,
+  and deleting options from the AST. The same interface is also provided
+  for the properties of an object expression.
+*/
+
+export function getProperty(r: ObjectExpression, label: string): Expression {
+  for (const prop of r.properties) {
+    if (
+      prop.key.type === 'Identifier' &&
+      'name' in prop.key &&
+      prop.key.name === label
+    ) {
+      return prop.value
+    }
+    if (
+      prop.key.type === 'StringLiteral' &&
+      'value' in prop.key &&
+      prop.key.value === label
```

Comment:
Is this a TypeScript thing where you have to check if this is a string? In my version of the api, I think this check is implicit my comparing it to the label.

Reply:
Or maybe I did this part wrong.

Reply:
Record labels can be both identifiers as well as string literals. The latter case is very rare but possible according to the AST.

Reply:
Yeah, that's a good catch. I'm not handling this case atm.

Reply:
I found that I only needed this check in setProperty. I'm not seeing the utility in getProperty.